python/src/deltachat/chatting.py

```diff
@@ -94,6 +129,34 @@ def mark_noticed(self):
         """
         return lib.dc_marknoticed_chat(self._dc_context, self.id)
 
+    # ------  group management API ------------------------------
+
+    def add_contact(self, contact):
+        """ add a contact to this chat.
+
+        :params: contact object.
+        :exception: ValueError if contact could not be added
+        :returns: None
+        """
+        ret = lib.dc_add_contact_to_chat(self._dc_context, self.id, contact.id)
+        if ret != 1:
+            raise ValueError("could not add contact {!r} to chat".format(contact))
+
+    def get_contacts(self):
+        """ get all contacts for this chat.
+
+        :params: contact object.
+        :exception: ValueError if contact could not be added
+        :returns: None
+        """
+        dc_array = ffi.gc(
+            lib.dc_get_contacts(self._dc_context, lib.DC_GCL_ADD_SELF, ffi.NULL),
```

Comment:
`dc_get_contacts()` returns all available contacts.
to get the contacts for a given chat, use dc_get_chat_contacts() - which takes the chat id as the second paramter then

Reply:
ok.  but wouldn't dc_get_chat_contacts() always return a subset of or the same as dc_get_contacts()?